core/src/main/java/nl/nn/adapterframework/core/IWrapperPipe.java

```diff
@@ -23,4 +23,5 @@
  */
 public interface IWrapperPipe extends IPipe {
 
+	enum Direction { WRAP, UNWRAP; };
```

Comment:
moet je deze niet expliciet public maken voor de frankdoc?

Reply:
Volgens mij hoeft dat niet in een interface